CHANGELOG.md

```diff
@@ -1,27 +1,34 @@
-﻿# Change Log
-
-<!--## Unreleased
-- Support GetCurrentWindowHandle and GetWindowHandles
-- Add driver option --silent to disable logging
--->
-
-
## v1.5.0
-
+﻿# Change Log
+
+<!--## Unreleased
+- Support GetCurrentWindowHandle and GetWindowHandles
+- Add driver option --silent to disable logging
+-->
+
+
+## v1.5.1
```

Comment:
@zelonght Unclear on this comment. Do you mean that we put it in Unreleased version without create new v1.5.1 in Changelog.md?